mermaid/er/examples_test.go

```
@@ -0,0 +1,143 @@
+		).
+		Relationship(
+			teachers,
+			schools,
+			er.OneToMoreRelationship,  // "|}"
+			er.ExactlyOneRelationship, // "||"
+			er.NonIdentifying,         // ".."
+			"School has many teachers",
+		).
+		String()
+
+	markdown.NewMarkdown(os.Stdout).
```

golangci-lint (1.62.2)

[low] G104: Errors unhandled.

(gosec)